scripts/requirements.txt

```diff
@@ -0,0 +1,2 @@
+PyYAML~=4.2b1
+mbed-tools~=7.3.3
```

Comment:
@cindyli-13 you can add ur new can stuff here soon

Reply:
I'm using pyyaml 5.1 rn, would that be an issue?

Reply:
shouldn't be but i can just uprev it anyways

Reply:
changed to 5.1